src/gluonts/evaluation/_base.py

```diff
@@ -99,6 +99,18 @@ class Evaluator:
         Default is 32.
     ignore_invalid_values
         Ignore `NaN` and `inf` values in the timeseries when calculating metrics.
+    aggregation_strategy:
+        Selects how invalid values in per-timeseries metrics should be filtered
+        when calculating the aggregate metric. Available options are:
+        `"all" | "valid" | "inf_only"`.
```

Comment:
Maybe mention here what the default is

Reply:
If this was a true strategy-pattern, the implementation would be in a different object. This implementation therefore also breaks the open-closed principle.

Reply:
There are cases in which a pre-defined set of options makes sense: this might be one?

Reply:
True that, updated it.